tutorials/0_to_litgpt.md

```diff
@@ -0,0 +1,549 @@
+  warnings.warn(*args, **kwargs)  # noqa: B028
+Missing logger folder: out/finetune/lora-phi-2/logs/csv
+Epoch 1 | iter 1 step 0 | loss train: 1.646, val: n/a | iter time: 820.31 ms
+Epoch 1 | iter 2 step 1 | loss train: 1.660, val: n/a | iter time: 548.72 ms (step)
+Epoch 1 | iter 3 step 1 | loss train: 1.687, val: n/a | iter time: 300.07 ms
+Epoch 1 | iter 4 step 2 | loss train: 1.597, val: n/a | iter time: 595.27 ms (step)
+Epoch 1 | iter 5 step 2 | loss train: 1.640, val: n/a | iter time: 260.75 ms
+Epoch 1 | iter 6 step 3 | loss train: 1.703, val: n/a | iter time: 568.22 ms (step)
+Epoch 1 | iter 7 step 3 | loss train: 1.678, val: n/a | iter time: 511.70 ms
+Epoch 1 | iter 8 step 4 | loss train: 1.741, val: n/a | iter time: 514.14 ms (step)
+Epoch 1 | iter 9 step 4 | loss train: 1.689, val: n/a | iter time: 423.59 ms
+Epoch 1 | iter 10 step 5 | loss train: 1.524, val: n/a | iter time: 603.03 ms (step)
+Training time: 11.20s
+Memory used: 13.90 GB
+Saving LoRA weights to 'out/finetune/lora-phi-2/final/lit_model.pth.lora'
+Saved merged weights to 'out/finetune/lora-phi-2/final/lit_model.pth'
+```
+
+Notice that the LoRA script saves both the LoRA weights (`'out/finetune/lora-phi-2/final/lit_model.pth.lora'`) and the LoRA weight merged back into the original model (`'out/finetune/lora-phi-2/final/lit_model.pth'`) for convenience. This allows us to use the finetuned model via the `chat` function directly:
+
+```bash
+litgpt chat --checkpoint_dir out/finetune/lora-phi-2/final/
+```
+
+```
+Now chatting with phi-2.
+To exit, press 'Enter' on an empty prompt.
+
+Seed set to 1234
+>> Prompt: Why are LLMs so useful?   
+>> Reply: LLMs are useful because they can be trained to perform various natural language tasks, such as language translation, text generation, and question-answering. They are also able to understand the context of the input data, which makes them particularly useful for tasks such as sentiment analysis and text summarization. Additionally, because LLMs can learn from large amounts of data, they are able to generalize well and perform well on new data.
+
+Time for inference: 2.15 sec total, 39.57 tokens/sec, 85 tokens
+
+>> Prompt: 
+```
+
+
+
+&nbsp;
+
+**More information and additional resources**
+
+- [tutorials/prepare_dataset](prepare_dataset): A summary of all out-of-the-box supported datasets in LitGPT and utilities for preparing custom datasets
+- [tutorials/finetune](finetune.md): An overview of the different finetuning methods supported in LitGPT
+- [tutorials/finetune_full](finetune_full.md): A tutorial on full-parameter finetuning
+- [tutorials/finetune_lora](finetune_lora.md): Options for parameter-efficient finetuning with LoRA and QLoRA
+- [tutorials/finetune_adapter](finetune_adapter.md): A description of the parameter-efficient Llama-Adapter methods supported in LitGPT 
+- [tutorials/oom](oom.md): Tips for dealing with out-of-memory (OOM) errors
+
+TODO: Mention config file resources
+
+&nbsp;
+## Inference (/chat)
```

Comment:
This is a strange title, It doesn't seem to fit with the others

Reply:
Changed it to LLM inference